tests/framework/vm.go

```diff
@@ -190,7 +190,7 @@ version: 2`
 			Name: vmName,
 		},
 		Spec: v1.VirtualMachineSpec{
-			Running: &no,
+			RunStrategy: ptr.To(v1.RunStrategyAlways),
```

Comment:
Shouldn't be `RunStrategyHalted`?

Reply:
yep! my bad.

Reply:
[DONE](https://github.com/kubevirt/kubevirt-velero-plugin/compare/65c2eb30d715daaf48f724226761145d38d2035d..20dde4fd2fd8d3dc7c0fc023fe1a0654e35012dd)